src/Avalonia.Diagnostics/Diagnostics/Screenshots/FilePickerHandler.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Avalonia.Controls;
+using Lifetimes = Avalonia.Controls.ApplicationLifetimes;
+
+namespace Avalonia.Diagnostics.Screenshots
+{
+    /// <summary>
+    /// Show a FileSavePicker to select where save screenshot
+    /// </summary>
+    public sealed class FilePickerHandler : BaseRenderToStreamHandler
+    {
+        /// <summary>
+        /// Get or sets the root folder where screeshots well be stored.
+        /// The default root folder is [Environment.SpecialFolder.MyPictures]/Screenshots.
+        /// </summary>
+        public string ScreenshotsRoot { get; set; }
```

Comment:
Can you make these properties a constructor parameters instead of mutable properties?
User will be able to instanciate it with custom parameters, easier than cast existing one.